saga-demo/saga-dubbo-demo/README.md

```diff
@@ -15,10 +15,88 @@ You will need:
 [alpha_server]: https://github.com/apache/incubator-servicecomb-saga/tree/master/alpha
 
 ## Running Demo
-```bash
-$ mvn clean install -DskipTests -Pdocker
-$ ./saga-dubbo-demo.sh up
+1. run the following command to create docker images in saga project root folder.
+   ```
+   mvn clean install -DskipTests -Pdocker -Pdemo
+   ```
+2. Enter the saga dubbo demo directory and give permissions to script
+   ```
+   cd ./saga-demo/saga-dubbo-demo
+   chmod +x saga-dubbo-demo.sh
```

Comment:
It can be fixed by add the +x in the sh file.  We don't need to instruct the user to do this thing.

Reply:
It has no executor right when first use,or shall we use like bash saga-dubbo-demo.sh?